fullstack/src/app/api/get-graph-data/route.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @api {get} /api/get-graph-data Get all data
+ * @description This file is the route for the get-graph-data API
+ */
+import { isRateLimited } from "@/app/utils/ratelimit";
+import db from "@/db";
+
+export const dynamic = 'force-dynamic';
+
+/**
+ * @description Gets a month worth of data of any type
+ */
+export const GET = async (request) => {
+    try {
+        const { searchParams } = new URL(request.url);
+        const ip = request.headers.get('x-forwarded-for') || request.connection.remoteAddress;
+        const MAX_REQUESTS = 7*15
```

Comment:
why not just say 105?

Reply:
Because if we add or remove data types it can just have the first value changed to match it